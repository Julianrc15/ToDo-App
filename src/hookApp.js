import React from 'react'

export const hookApp = () => {
  return (

    <div>
        <h1>
        Hooks-app
        </h1>

    </div>
  )
}
