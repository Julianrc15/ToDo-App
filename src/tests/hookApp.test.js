import { shallow } from 'enzyme';
import React from 'react';
import '@testing-library/jest-dom';
import { hookApp } from '../hookApp';



describe('Pruebas en hookApp',()=> {

const wrapper = shallow(<hookApp/>);

test('Snapshot debe coincidir', () => {

    expect ( wrapper ).toMatchSnapshot();
  
});


})