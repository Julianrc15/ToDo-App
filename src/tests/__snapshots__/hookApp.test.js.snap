// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en hookApp Snapshot debe coincidir 1`] = `<hookApp />`;
